### ISSUE TITLE
What does cls_layer do when cls_type is Linear?

### ISSUE THREAD

**Issue by deep-practice** [JDAI-CV/fast-reid#502]
cls_outputs = self.cls_layer(logits.clone(), targets)

**Comment by L1aoXingyu** [comment]
Check here
https://github.com/JDAI-CV/fast-reid/blob/master/fastreid/layers/any_softmax.py

**Comment by deep-practice** [comment]
Thxs.
